3rdparty/lua-5.1.3/src/linit.c

```diff
@@ -19,7 +19,6 @@ static const luaL_Reg lualibs[] = {
   {LUA_LOADLIBNAME, luaopen_package},
   {LUA_TABLIBNAME, luaopen_table},
   {LUA_IOLIBNAME, luaopen_io},
-  {LUA_OSLIBNAME, luaopen_os},
```

Comment:
I guess we actually do need to do this.  :hocho: :moon:  tvOS won't let you use any system commands.  :tv: :hand: